src/configure/configure.ts

```diff
@@ -245,7 +292,15 @@ class PipelineConfigurer {
                 title: utils.format(Messages.creatingAzureServiceConnection, this.inputs.targetResource.subscriptionId)
             },
             () => {
-                return this.serviceConnectionHelper.createAzureServiceConnection(this.inputs.targetResource.resource.name, this.inputs.azureSession.tenantId, this.inputs.targetResource.subscriptionId);
+                let scope = this.inputs.targetResource.resource.id;
```

Comment:
should it be a function in some helper ? so that it is easy to get scope for webapp on containers or AKS by finding Least common ancestor in the resourceId ?
The function can accept targetResource and an array of other resources.

Reply:
I didn't find an appropriate place to put this actually. we can take it up when we do the grow up scenario. Maybe each target type can implement a method getScope which can return as per the requirement. Skipping it now.